app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user, index: { unique: true }, foreign_key: true
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    user.increment!(:posts_counter)
+  end
+
+  def recent_five_comments
+    comments.order!(created_at: :desc).limit(5)
+  end
+end
```

Comment:
- [ ] You have a problem with your migration, kindly take a step back and update it
![image](https://user-images.githubusercontent.com/62705947/196985200-db4a40ed-7a14-40db-a957-2c40f63406d6.png)

Reply:
Hi, @GzimAsani thanks for the review. great catch. I tried to re-generate the migrate files and it worked fine but In the console to create posts, it is giving me an error as in the figure. I couldn't figure it out, could you help me?
![Screenshot from 2022-10-20 21-24-53](https://user-images.githubusercontent.com/75738563/197029873-b011e787-28aa-4537-ad65-81e80464ff28.png)